code/controllers/subsystem/minor_mapping.dm

```diff
@@ -16,26 +16,20 @@ SUBSYSTEM_DEF(minor_mapping)
 /datum/controller/subsystem/minor_mapping/proc/trigger_migration(num_mice=10)
 	var/list/exposed_wires = find_exposed_wires()
 
-	var/mob/living/basic/mouse/mouse
 	var/turf/open/proposed_turf
-
-
 	while((num_mice > 0) && exposed_wires.len)
 		proposed_turf = pick_n_take(exposed_wires)
-
-		if(!istype(proposed_turf))
+		if (!istype(proposed_turf)||
+			!proposed_turf.air.has_gas(/datum/gas/oxygen, 5) ||
+			proposed_turf.temperature > NPC_DEFAULT_MAX_TEMP ||
+			proposed_turf.temperature < NPC_DEFAULT_MIN_TEMP)
 			continue
```

Comment:
```suggestion
		if (!istype(proposed_turf) || \
			!proposed_turf.air.has_gas(/datum/gas/oxygen, 5) || \
			proposed_turf.temperature > NPC_DEFAULT_MAX_TEMP || \
			proposed_turf.temperature < NPC_DEFAULT_MIN_TEMP)
			continue
```

Reply:
I think byond doesn't know how to parse `... || \n`

Reply:
Does it need the backslashes? If statements can natively support being multilined, I thought those should only be required if it was a macro.